requirements.txt

```diff
@@ -11,3 +11,4 @@ Flask >= 0.10.1
 iso8601 >= 0.1.11
 resultsdb_api >= 2.0
 six >= 1.10.0
+cachelib
```

Comment:
Why this is needed?

Reply:
It is used in `resultsdb_frontend/controllers/main.py`. My guess is that previously it was installed as a dependency of a library mentioned in `requirements.txt` transitively.